editor/editor_inspector.cpp

```diff
@@ -3074,7 +3074,7 @@ void EditorInspector::update_tree() {
 					editor_inspector_array = memnew(EditorInspectorArray(all_read_only));
 					int page = per_array_page.has(array_element_prefix) ? per_array_page[array_element_prefix] : 0;
 
-					editor_inspector_array->setup_with_count_property(object, class_name_components[0], p.name, array_element_prefix, page, c, foldable, movable, numbered, page_size, add_button_text, swap_method);
+					editor_inspector_array->setup_with_count_property(object, class_name_components[0], p.name, array_element_prefix, page, c, foldable, movable, numbered, page_size, TTR(add_button_text), swap_method);
```

Comment:
I think the TTR() should be at the "Add Element" string, so the text can be fetched for translation. The button will be auto-translated anyway (I think).

Reply:
Fixed.